src/java/com/github/jsdossier/CompilerModule.java

```diff
@@ -78,7 +79,7 @@ CompilerOptions provideCompilerOptions(
 
     // IDE mode must be enabled or all of the jsdoc info will be stripped from the AST.
     options.setIdeMode(true);
-    options.setPreserveJsDocWhitespace(true);
+    options.setParseJsDocDocumentation(Config.JsDocParsing.INCLUDE_DESCRIPTIONS_WITH_WHITESPACE);
```

Comment:
That would require an release from the js compiler team. This may or may not be non-trivial. 

@blickly WDYT?

My main priority is to get this fixed internally. Our documentation push is blocked atm. Perhaps a more expedient solution would be to stop building the internal binary from compiler head but from an old release jar.